packages/terra-core-docs/src/terra-dev-site/doc/text/example/TextTime.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Text from 'terra-text';
+import moment from 'moment';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import styles from '../../../../../../terra-text/src/Text.module.scss';
+
+const cx = classNamesBind.bind(styles);
+const descriptiveClasses = classNames(
+  cx([
+    'text',
+    { 'visually-hidden': true },
+  ]),
+);
+
+const timeVar = moment('10:22:30', 'hh:mm:ss').format('hh:mm:ss');
+
+const timeParts = timeVar.split(':');
+const hourVal = timeParts[0];
+const minuteVal = timeParts[1];
+const secondVal = timeParts[2];
+
+const valueToDisplay = (
+  <>
+    <span className={descriptiveClasses}>Hour</span>
```

Comment:
is there any specific reason for having this classname on labels ??

Reply:
This adds the necessary styling to the label/span so as to not visually show it on the screen, and instead only be perceptible to the screen readers.